package.json

```diff
@@ -80,7 +80,7 @@
     "babel-plugin-transform-es2015-template-literals": "^6.22.0",
     "babel-plugin-transform-proto-to-assign": "^6.23.0",
     "babel-template": "^6.24.1",
-    "backburner.js": "^1.2.2",
+    "backburner.js": "1.2.3",
```

Comment:
Pinned to exact version to avoid getting `backburner.js@1.3.0` (which contains other changes that should flow through a full canary / beta cycle) in beta.  

See https://github.com/emberjs/ember.js/pull/15874 for that bump (canary only).